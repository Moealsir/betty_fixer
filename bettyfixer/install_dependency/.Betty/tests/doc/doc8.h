#ifndef _DOC6_H_
#define _DOC6_H_

/**
 * dog - A Dog structure
 *
 * @name: Name
 * @age: Test
 * @owner: The owner
 */
struct dog
{
	char *name;
	float age;
	char *owner;
};

/**
 * test - Test
 *
 * @FIRST: First
 * @SECOND: Second
 * @THIRD: Third
 */
enum test
{
	FIRST = 0,
	SECOND,
	THIRD
};

/**
 * color - Color
 *
 * @rgba: RGBA
 * @value: Value
 */
union color
{
	char rgba[4];
	unsigned int value;
};

/**
 * animal - Animal
 *
 * @name: The name
 * @race: Race
 * @color: color
 */
typedef struct animal
{
	char *name;
	enum test race;
	union color color;
} t_animal;

/**
 * test2 - Test
 *
 * @FIRST: First
 * @SECOND: Second
 * @THIRD: Third
 */
typedef enum test2
{
	FIRST = 0,
	SECOND,
	THIRD
} e_test2;

/**
 * color2 - Color
 *
 * @rgba: RGBA
 * @value: Value
 */
typedef union color2
{
	char rgba[4];
	unsigned int value;
} u_color;

/* No doc is required for func prototypes */
void init_dog(struct dog *d, char *name, float age, char *owner);
void print_dog(struct dog *d);

#endif /* _DOC6_H_ *?
