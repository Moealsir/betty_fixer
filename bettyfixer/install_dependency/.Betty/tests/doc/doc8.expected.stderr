tests/doc/doc8.h:11: warning: no description found for struct dog
tests/doc/doc8.h:25: warning: no description found for enum test
tests/doc/doc8.h:38: warning: no description found for union color
tests/doc/doc8.h:51: warning: no description found for struct animal
tests/doc/doc8.h:65: warning: no description found for enum test2
tests/doc/doc8.h:78: warning: no description found for union color2
